src/v2/ion/ViewClassNamesFactory.js

```diff
@@ -24,7 +24,6 @@ const FORMNAME_CLASSNAME_MAPPINGS = {
     [AUTHENTICATOR_KEY.OV]: 'mfa-verify',
     [AUTHENTICATOR_KEY.GOOGLE_OTP]: 'mfa-verify',
     [AUTHENTICATOR_KEY.DUO]: 'mfa-verify-duo',
-    [AUTHENTICATOR_KEY.SYMANTEC_VIP]: 'mfa-symantec',
```

Comment:
why was this removed? This would be `mfa-verify` right?

Reply:
The `mfa-symantec` class is unfortunately mapped to CSS that adds a [background image](https://github.com/okta/okta-signin-widget/blob/master/assets/sass/okta-theme.scss#L285-L295). Will change this to `mfa-verify`!